te_markdown/terminology.md

```diff
@@ -46,13 +46,13 @@ In this area, details about the test step selected in the _Test Execution Naviga
 <a name="application-mapping-language"></a>Application Mapping Language (AML)
 : A language used by [testers](#tester) to define the platform- or application-specific vocabulary for the [test steps](#test-step), and by [developers](#developers) to map these steps to the concrete actions to be performed for testing, implemented in [test fixtures](#test-fixture).
 
-<a name="domain-expert"></a>Domain Expert
-: A user of the [software under test](#software-under-test), or other stakeholder, who has a deep understanding of the problem domain. Domain experts specify what they expect the system to do and how it should behave, using the [test specification language (TSL)](#test-specification-language).
-
 <a name="developer"></a>Developer
 : A person who is involved in the development of the [software under test](#software-under-test). User acceptance tests described with the [test case language (TCL)](#test-case-language) are continuously executed during the development process to check the progress with each development increment, and to validate the implementation.
 
-<a name="software-Under-Test"></a>Software Under Test (SUT)
+<a name="domain-expert"></a>Domain Expert
+: A user of the [software under test](#software-under-test), or other stakeholder, who has a deep understanding of the problem domain. Domain experts specify what they expect the system to do and how it should behave, using the [test specification language (TSL)](#test-specification-language).
```

Comment:
```suggestion
: A user of the [Software Under Test](#software-under-test), or other stakeholder, who has a deep understanding of the problem domain. Domain experts specify what they expect the system to do and how it should behave, using the [test specification language (TSL)](#test-specification-language).
```

Reply:
I don't believe you would capitalize this, its not a proper name – is it?